writing/house-style.md

```diff
@@ -0,0 +1,60 @@
+    * [Tone of voice](#tone-of-voice)
+    * [Plain English](#plain-english)
+    * [Technical writing](#technical-writing)
+* [Frontend or front-end](#frontend-or-front-end)
+
+
+## Writing style
+
+We don't enforce strict standards on writing style as we want you to use your own words to express your own personality. That said, we have three general sets of guidelines - tone of voice, plain English, and technical writing.
+
+
+### Tone of voice
+
+For a general audience, you should write like a human. Be professional, but conversational - we want our contributors and readers to feel comfortable joining in at any time. The exception to this is in writing [technical content](#technical-writing) for a technical audience.
+
+Be thankful to contributors and those who log issues. They're giving up their time to make our playbook better. Make sure they know we're grateful to them.
+
+Maintain the same friendly and professional standards across the entire repository - in documentation, issues, pull requests, and comments.
+
+Always use [inclusive language](inclusive-language.md).
+
+
+### Plain English
+
+Follow Plain English guidelines to help safeguard the accessibility of your written content. All relevant guidelines for [writing well][writing-well] apply. You might find tools like [Hemingway App][hemingway] helpful when editing. In particular:
+
+- Be concise  
+  - Check any sentences with more than 25 words to see if you can split them to make them clearer.
+  - Use contractions (e.g. "can't", "it's").
+  - Go straight to the point.
+- Be specific
+  - Use the active voice instead of the passive voice (e.g. "NPM will install the dependencies" instead of "the dependencies will be installed").
+  - Address the user as "you" where possible.
+  - Avoid referring to "things" or "stuff".
+  - Be direct. Avoid metaphor, simile, and slang.
+- Be informative
+  - Consider your audience. General information may not look the same as information aimed at a technical audience.
+  - Be as precise as you can usefully be - consider linking to high quality external resources where relevant.
+
+
+### Technical writing
+
+When expressing technical concepts, you'll need to use technical language. This may be complex, contain jargon, or seem formal. That's ok! Technical writing isn't for a general audience. Your priorities must be clarity, brevity, and accuracy, not the emotional impact of your words. 
+
+All of the advice in the [plain English](#plain-english) section applies. Additionally:
+
+- Expand acronyms the first time you use them, and limit their use thereafter e.g. "Avoid the use of TLAs (Three Letter Acronyms)".
+- When defining technical requirements, you may use the keywords defined in [RFC 2119](https://www.ietf.org/rfc/rfc2119.txt) for those requirement levels, e.g. "MUST", "SHOULD", "OPTIONAL".
+
+
+## Frontend or front-end
+
+* Use `front-end` when you need an adjective (e.g. "the front-end team", "front-end principles").
```

Comment:
Should Front-End Team be capitalised, as it's a specific thing?

Reply:
A _certain contributor_ 👀 noted in #188 that "... arguably it doesn't need to be in there - the most important thing with this is how we use it when naming things like repos..." which I don't disagree with. 

and I disagree with it even less when i think about going through the playbook and recapitalising every instance of "front-end team" 😂

Reply:
Soz, the purpose of my comment is unclear, isn't it?

To clarify, I'm objecting to changing the capitalisation on the grounds that the entire sentence/practice is unnecessary. Prescriptive enforcement of the two different types of use is a complication that IMO we'd be better off without. 

I'd prefer to sack the first instruction off entirely and only be prescriptive when it comes to repo names/slack channels/code.

 ¯\_(ツ)_/¯

Reply:
When it's used as a noun i.e. proper noun then that makes sense as capitalised.

Otherwise in common usage or as an adjective (where you are just qualifying another common noun) then lower case is fine.